components/infra-proxy-service/server/util.go

```diff
@@ -39,3 +39,38 @@ func SafeSliceFromMap(values map[string]interface{}, key string) []string {
 
 	return []string{}
 }
+
+// SubstractSlice return the slice without containing all element of b.
+func SubstractSlice(a []string, b []string) []string {
```

Comment:
Fix typo and make the explanation more clear.
```suggestion
// SubtractSlice returns the slice a with all elements of b removed.
func SubtractSlice(a []string, b []string) []string {
```